### ISSUE TITLE
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR on agent startup

### ISSUE THREAD

**Issue by cmaliwal** [fetchai/uAgents#479]
### Prerequisites

- [X] I checked the [documentation](https://fetch.ai/docs) and found no answer to my problem
- [X] I checked the [existing issues](https://github.com/fetchai/uAgents/issues) and made sure there are no similar bug reports

### Category

Bug (unexpected behavior)

### Expected Behavior

The agent should start without displaying any unexpected warnings or log messages related to absl::InitializeLog().

### Observed Behavior

Upon starting the agent, the following warning is displayed in the console:

```
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1722521954.276588 2335542 config.cc:230] gRPC experiments enabled: call_status_override_on_cancellation, event_engine_dns, event_engine_listener, http2_stats_fix, monitoring_experiment, pick_first_new, trace_record_callops, work_serializer_clears_time_cache
I0000 00:00:1722521954.390933 2335542 work_stealing_thread_pool.cc:320] WorkStealingThreadPoolImpl::PrepareFork
```

### Version

0.14.0

### Environment Details (Optional)

```markdown
Operating System: macOS
```

**Comment by jrriehl** [comment]
Looks like this will be fixed by updating `grpcio`.

**Comment by Archento** [comment]
Fixed by #487
